### ISSUE TITLE
Can't get uri of photo on ios

### ISSUE THREAD

**Issue by RomanKameniev** [react-native-community/react-native-cameraroll#135]
Hi there.
I tried to use cameraroll to display pictures from phone in my application and  give user ability to select photo from my flat list. 
For now i taking name and uri, but uri have format ph//0F4DC998-56ED-48AD-B503-BC418537A4FD 
How can i transform this uri to valid format and display photo in my application?

**Comment by xKRUTOIx** [comment]
Hello, it's weird because I was able to display images in FlatList on iOS with ph-format. Although I couldn't use this ph-path to send it to a server.

Anyway, I've solved this problem with this module – https://github.com/itinance/react-native-fs

Take a look at this function:
![image](https://user-images.githubusercontent.com/6790512/73154235-bc5c2580-40e7-11ea-88c6-2beacacc6773.png)

If it won't help then [this issue](https://github.com/react-native-community/react-native-cameraroll/issues/52) might help.
